src/screens/login/LoginScreen.view.js

```diff
@@ -56,6 +56,7 @@ export const LoginScreenView = ({
           <ActivityIndicator size={'large'} color={color.primary} />
```

Comment:
We already have activity indicator in container. We don't need it here.

I'm from mobile but will take a look from laptop as well. Let's other folks to take a look

Reply:
Our solution is to create a new story/PR for using the ActivityIndicator from Container for everything that we forgot.